src/app/urls.py

```diff
@@ -38,6 +38,7 @@
     url(r'^checkusername/$', views.checkusername, name='check-username'),
     url(r'^xml_upload/$',views.xml_upload, name='xml-upload'),
     url(r'^edit/(?P<page_id>[0-9a-z]+)/$', views.xml_edit, name='editor'),
+    url(r'^edit_license_xml/(?P<license_id>[0-9]+)/$', views.edit_license_xml, name='license_xml_editor'),
```

Comment:
I would suggest you to put `*` instead of `+` at that Regex, then only the redirect would work

Reply:
@techytushar 
I have added the changes you kindly requested.